tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -143,6 +145,129 @@ def test_fail_on_multiple_drivers():
         extract_db_type_from_uri("mysql+pymsql+pyodbc://...")
 
 
+@dataclass
+class TestDatabase:
+    db_url: str
+    store: SqlAlchemyStore
+
+
+@pytest.fixture
+def test_database(tmp_path: Path) -> TestDatabase:
+    db_url = _setup_db_uri(tmp_path)
+    store = _get_store(db_url)
+    yield TestDatabase(db_url, store)
```

Comment:
Can we just yield `store`?

Reply:
Yes, I made it here just to return db_uri and directly pass tmp_path to the actual test methods.